src/nft/components/bag/profile/ListingModal.tsx

```diff
@@ -139,98 +173,106 @@ const ListingModal = () => {
   const clickStartListingFlow = () => {
     resetAllRows()
     allCollectionsApproved ? signListings() : startListingFlow()
+    !allCollectionsApproved && sendAnalyticsEvent(EventName.NFT_SELL_START_LISTING, { ...startListingEventProperties })
   }
 
   const showSuccessScreen = useMemo(() => listingStatus === ListingStatus.APPROVED, [listingStatus])
 
   return (
-    <Column paddingTop="20" paddingBottom="20" paddingLeft="12" paddingRight="12">
-      <Row className={headlineSmall} marginBottom="10">
-        {isMobile && !showSuccessScreen && (
-          <Box paddingTop="4" marginRight="4" onClick={toggleCart}>
-            <ChevronLeftIcon height={28} width={28} />
+    <Trace modal={ModalName.NFT_LISTING}>
+      <Column paddingTop="20" paddingBottom="20" paddingLeft="12" paddingRight="12">
+        <Row className={headlineSmall} marginBottom="10">
+          {isMobile && !showSuccessScreen && (
+            <Box paddingTop="4" marginRight="4" onClick={toggleCart}>
+              <ChevronLeftIcon height={28} width={28} />
+            </Box>
+          )}
+          {showSuccessScreen ? 'Success!' : `Listing ${sellAssets.length} NFTs`}
+          <Box
+            as="button"
+            border="none"
+            color="textSecondary"
+            backgroundColor="backgroundSurface"
+            marginLeft="auto"
+            marginRight="0"
+            paddingRight="0"
+            display={{ sm: 'flex', md: 'none' }}
+            cursor="pointer"
+            onClick={toggleCart}
+          >
+            <XMarkIcon height={28} width={28} fill={themeVars.colors.textPrimary} />
           </Box>
-        )}
-        {showSuccessScreen ? 'Success!' : `Listing ${sellAssets.length} NFTs`}
-        <Box
-          as="button"
-          border="none"
-          color="textSecondary"
-          backgroundColor="backgroundSurface"
-          marginLeft="auto"
-          marginRight="0"
-          paddingRight="0"
-          display={{ sm: 'flex', md: 'none' }}
-          cursor="pointer"
-          onClick={toggleCart}
-        >
-          <XMarkIcon height={28} width={28} fill={themeVars.colors.textPrimary} />
-        </Box>
-      </Row>
-      <Column overflowX="hidden" overflowY="auto" style={{ maxHeight: '60vh' }}>
+        </Row>
+        <Column overflowX="hidden" overflowY="auto" style={{ maxHeight: '60vh' }}>
+          {showSuccessScreen ? (
+            <Trace
+              name={EventName.NFT_LISTING_COMPLETED}
```

Comment:
Currently this only logs when ALL listings have succeeded, do we want an event for each successful listing as well?